src/renderer/pages/torrent-list-page.js

```diff
@@ -1,5 +1,6 @@
 const React = require('react')
 const prettyBytes = require('prettier-bytes')
+const natsort = require('natsort')
```

Comment:
What do you think about using javascript's native `localeCompare` instead an external library?